README.md

```diff
@@ -16,6 +16,7 @@ The buildpack will do the following:
 * Requests that the Native Image builder be installed by requiring `native-image-builder` in the build plan.
 * If `$BP_BINARY_COMPRESSION_METHOD` is set to `upx`, requests that UPX be installed by requiring `upx` in the buildplan.
 * Uses `native-image` a to build a GraalVM native image and removes existing bytecode. Defaults to building the `/workspace` as an exploded JAR. If `$BP_NATIVE_IMAGE_BUILT_ARTIFACT` is set, it will build from the specified JAR file.
+* If a native-image argument file is provided by Spring-Boot at `META-INF/native-image/argfile`, this file will be appended to the arguments of the `native-image` command using the `@argfile` format.
```

Comment:
I don't think this needs to reference Spring Boot, as any framework could choose to use this behavior to provide arguments in a similar way.

Reply:
Thanks Scott!